### ISSUE TITLE
Bug: Elasticsearch 5.6 removed from Homebrew

### ISSUE THREAD

**Issue by Leland** [weprovide/valet-plus#513]
- [x] I've checked the issue queue and could not find anything similar to my bug.
- [x] I'm on the latest version of valet-plus (`valet --version`): `Valet+ 2.0.0`
- [x] I've run `valet fix` and `valet install` after updating and before submitting my issue/feature.

**What is the problem?**
```
$ valet use elasticsearch 5.6 
[elasticsearch@5.6] Installing
Error: No available formula with the name "elasticsearch@5.6" 
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

elasticsearch@5.6 was deleted from homebrew/core in commit 157167491:
  elasticsearch and kibana: switched to major versions and removed old versions
  Closes https://github.com/Homebrew/homebrew-core/issues/57875.
  Signed-off-by: Sean Molenaar <1484494+SMillerDev@users.noreply.github.com>

To show the formula before removal run:
  git -C "$(brew --repo homebrew/core)" show 157167491^:Formula/elasticsearch@5.6.rb

If you still use this formula consider creating your own tap:
  https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap


In Brew.php line 108:
                                                   
  Brew was unable to install [elasticsearch@5.6].  
                                                   

use [<service>] [<targetVersion>]
```

**What was supposed to happen?**
Elasticsearch 5.6 should be installed

**What actually happened?**
Errored

**Comment by fpurser** [comment]
I've also encountered this trying to run Magento 2.4.0 locally.

**Comment by nickolasburr** [comment]
@fpurser Magento 2.4.0 requires Elasticsearch 7.6. Unfortunately, none of the previous versions are compatible and, to make matters worse, the latest version (7.8.1) is not yet supported, either. I had to find the version of the formula when 7.6.2 was the latest release and download it directly:

`brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/1f6d3894/Formula/elasticsearch.rb`

**Comment by Neodork** [comment]
Hey, we've solved this in the new release 2.1.0. Closing the ticket.
